configure.ac

```diff
@@ -246,22 +244,26 @@ AS_IF([test "x$with_docs_only" = xyes],
      AC_ARG_ENABLE([params-check],
          AS_HELP_STRING([--disable-params-check],
                         [Disable checking user parameters passed to API, default: NO]),
-         [AC_DEFINE([ENABLE_PARAMS_CHECK], [0])],
-         [AC_DEFINE([ENABLE_PARAMS_CHECK], [1], [Enable checking user parameters])])
-
+         [], 
+         [enable_params_check=yes])
+     AS_IF([test "x$enable_param_check" = xyes],
+     	   [AC_DEFINE([ENABLE_PARAMS_CHECK], [0])],
```

Comment:
it should be other way around - 1 (probably this caused test failure)

Reply:
Ouch